analyzeDelphes.py

```diff
@@ -6,10 +6,11 @@
 particle={
     11:"e",
     13:"mu",
-    21:"gamma", #using this for all jets
-    22:"j",
+    21:"j", #using this for all jets
+    22:"gamma",
     23:"z",
-    24:"w"
```

Comment:
I think you should add ptMin and etaMax cuts below for 23 and 24, even if they are zero/infinity